docs/core/whats-new/index.md

```diff
@@ -3,12 +3,14 @@ title: What's new in .NET Core
 description: Learn about the new features found in .NET Core.
 author: rpetrusha
 ms.author: ronpet
-ms.date: 05/07/2018
+ms.date: 12/4/2018
 ---
 # What's new in .NET Core
 
 This page provides a summary of new features in each release of .NET Core starting with .NET Core 2.0. The following links provide detailed information on the major features added in each release.
 
+- [.NET Core 2.2](dotnet-core-2-2.md
+
 - [.NET Core 2.1](dotnet-core-2-1.md)
 
 - .[NET Core 2.0](dotnet-core-2-0.md)
```

Comment:
Not related nit: the dot should be included into the link text as for the above two links.